doc/content/reference/telemetry/cli.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "CLI Telemetry"
+description: "Description of everything that the telemetry procedure collects regarding the CLI's user"
+aliases: "/telemetry/cli"
+---
+
+The Things Stack\'s [CLI]({{<ref "/reference/cli">}}) collects **completely anonymous** telemetry data, which is used to help us understand how users interact with our product. For example, the current data helps to have a better insight on what type of machines the CLI is more commonly used. The objective with the introduction of telemetry is to be able to better understand what is used so that we can expand and improve on the experience that we provide to our users.
+
+## Identification
+
+The worry that data is attached to an easily identifiable piece of information is common and for good reason, therefore we believe that it is the first thing we should address.
+
+When the CLI start the procedure to collect and send the telemetry data for the first time, it creates a file on the system's cache folder which contains the following data: 
```

Comment:
`When the CLI starts for the first time on a new machine or upon clearing the system cache, it creates a file on the system's cache folder with the following data`.